hbase-shell/src/main/ruby/shell/commands/deleteall.rb

```diff
@@ -30,12 +30,16 @@ def help
   hbase> deleteall 't1', 'r1'
   hbase> deleteall 't1', 'r1', 'c1'
   hbase> deleteall 't1', 'r1', 'c1', ts1
+  //'' means no specific column, will deletes all cells in the row which timestamp is lower than
```

Comment:
Sorry for being pedant, still has an english grammar error. Should be:

> //'' means no specific column, will delete all cells in the row which timestamp is lower than

Reply:
fix, again... thanks a lot.